### PR TITLE
Created design for the join game page

### PR DESCRIPTION
**Pull request by JulieL999**

### CODE REVIEW

**Comment by dapopovic** [comment]
join game class does not exist

File: Client/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -26,6 +26,9 @@
             android:name=".Lobby"
             android:exported="false"
             android:screenOrientation="sensorLandscape"/>
+        <activity android:name=".Join_Game"
+            android:exported="true"
```
Comment (dapopovic) [inline Client/app/src/main/AndroidManifest.xml:30]: this should be false

**Review by Ragnarus — COMMENTED** [review]
You could merge it

File: Server/src/test/java/at/aau/anti_mon/server/websocket/WebSocketHandlerClientImpl.java [hunk 2]
```diff
@@ -1,37 +1,80 @@
...
+            try {
+
+                JsonDataDTO  data = JsonDataManager.parseJsonMessage(json);
+                Commands commands = data.getCommand();
 
+                Logger.info("CLIENT : Command: " + commands.getCommand());
+                Logger.info("CLIENT : Data: " + data);
+
+                switch (commands) {
+                    case ANSWER -> {
```
Comment (dapopovic) [inline Server/src/test/java/at/aau/anti_mon/server/websocket/WebSocketHandlerClientImpl.java:49]: Case Pin is missing

**Review by dapopovic — CHANGES_REQUESTED** [review]

**Pull request merged**